website/docs/concepts/infrastructure.md

```diff
@@ -14,6 +14,14 @@ With the latest release of LitmusChaos 3.0.0 the term **Chaos Delegate/Agent** h
 
 An environment represents where you are installing your chaos infrastructure and acts as an additional level of abstraction for the same. You categorize each environment as prod or non-prod.
 
+## Types of Chaos Infrastructures
+
+In Litmus, chaos infrastructures can be classified into two types:
+
+- **Self Chaos Infrastructures:** A Chaos Infrastructure that is connected to the same cluster and namespace where the ChaosCenter is deployed. It can be used to target the workloads executing on that cluster only.
+
+- **External Chaos Infrastructures:** A Chaos Infrastructure that is connected to a remote Kubernetes cluster. ChaosCenter can be operated in a cross-cloud manner, which allows connecting multiple External Chaos Infrastructure to the same ChaosCenter with the help of the [litmusctl](../litmusctl/installation.md) CLI. Once connected you can manage, monitor, observe and induce chaos from the ChaosCenter to the respective External Chaos Infrastructures.
+
```

Comment:
Since we no longer divide our infrastructure into self and external, can we delete that part? Sorry for making you do the same thing.

Reply:
Done